src/utils/create-entity-api-utils.ts

```diff
@@ -32,7 +41,22 @@ export const createEntityApiUtils = <
   >;
   const entityName = api.reducerPath;
 
+  /**
```

Comment:
Shouldn't all the documentation below be in `src/types/entity-api-utils.ts`?

@ipakhomov what do you think?